test/dummy/lib/design_system/components/layout/layout_facade.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class LayoutFacade < FenrirView::Presenter
+  property :title
+
+  # Layout section handling
+  attr_reader :layout
+
+  def layout=(context)
+    @layout = LayoutSectionCollector.new(context: context)
+  end
+
+  def layout_sections
```

Comment:
Is this valid sections? Could be a constant on the class `VALID_LAYOUT_SECTIONS=%i[...]` as it doesn't change per instance